doozer/doozerlib/cli/scan_osh.py

```diff
@@ -3,50 +3,41 @@
 from doozerlib.cli import cli, click_coroutine, pass_runtime
 from doozerlib.runtime import Runtime
 from doozerlib.exectools import fire_and_forget
-from doozerlib.constants import BREWHUB_URL
 from doozerlib.util import cprint
 
 
 class ScanOshCli:
-    def __init__(self, runtime: Runtime, brew_tags: list, last_brew_event):
+    def __init__(self, runtime: Runtime, last_brew_event: int, dry_run: bool):
         self.runtime = runtime
-        self.brew_tags = brew_tags
         self.last_brew_event = last_brew_event
+        self.dry_run = dry_run
+        self.brew_tags = []
 
-        # Initialize runtime
-        self.runtime.initialize(clone_distgits=False, no_group=True)
+        # Initialize runtime and brewhub session
+        self.runtime.initialize(clone_distgits=False)
+        self.koji_session = koji.ClientSession(self.runtime.group_config.urls.brewhub)
 
-    @staticmethod
-    async def get_tagged_latest(tag):
+    async def get_tagged_latest(self, tag):
         """
         Returns the latest RPMs and builds tagged in to the candidate tag received as input
         """
-        session = koji.ClientSession(BREWHUB_URL)
-        try:
-            latest_tagged = session.listTagged(tag=tag, latest=True)
-            if latest_tagged:
-                return latest_tagged
-        except Exception:
-            raise
-
-    @staticmethod
-    async def get_tagged_all(tag):
+        latest_tagged = self.koji_session.listTagged(tag=tag, latest=True)
+        if latest_tagged:
+            return latest_tagged
+
```

Comment:
We could return [] instead of None when there's no latest_tagged?

Reply:
done, fixed in https://github.com/openshift-eng/art-tools/pull/51/commits/c4c66665b3c5b9cc9d9b473fb0af3c5dbdf1253c